libmamba/include/mamba/api/remove.hpp

```diff
@@ -18,11 +18,18 @@ namespace mamba
     class ChannelContext;
     class Configuration;
 
-    void remove(Configuration& config, int flags = MAMBA_REMOVE_PRUNE);
+    enum RemoveResult
+    {
+        YES = 0,
+        NO = 1,
+        EMPTY = 2,
+    };
```

Comment:
Prefer using:
```suggestion
    enum class RemoveResult : int
    {
        YES = 0,
        NO = 1,
        EMPTY = 2,
    };
```
to avoid implicit conversions.